comfy_cli/command/distribution.py

```
@@ -882,8 +1139,41 @@ def _create_execute(renderer, definition: dict, *, name: str, builder_url: str |
+            renderer.warn(line)
+        checked = (imported.get("definition") or {}).get("customNodes")
+        if checked is not None:
+            # A pack it could not vouch for is absent from what it returns, and
+            # an image quietly missing a pack is worse than a refused create.
+            kept = {n.get("name") for n in checked}
+            dropped = [n for n in declared if n not in kept]
+            if dropped:
+                renderer.error(
+                    code="distribution_registry_pin_missing",
+                    message="the builder could not resolve these custom nodes: " + ", ".join(sorted(dropped)),
+                    hint="see the advisories above; edit the definition to name a published version, or remove the pack",
```

Pylint (4.0.6)

[convention] Line too long (121/120)

(C0301)